web/portal/brand/src/entities/Retail/Form.tsx

```diff
@@ -47,39 +85,48 @@ const Form = (props: EntityFormProps): JSX.Element => {
       legend: _('Retail specific'),
       fields: ['routingTagIds', 'codecIds'],
     },
-    edit && {
-      legend: _('Invoice data'),
-      fields: [
-        'showInvoices',
-        'invoicing.nif',
-        'invoicing.postalAddress',
-        'invoicing.postalCode',
-        'invoicing.town',
-        'invoicing.province',
-        'invoicing.countryName',
-      ],
-    },
+    edit &&
+      hasInvoicesFeature && {
+        legend: _('Invoice data'),
+        fields: [
+          'showInvoices',
+          'invoicing.nif',
+          'invoicing.postalAddress',
+          'invoicing.postalCode',
+          'invoicing.town',
+          'invoicing.province',
+          'invoicing.countryName',
+        ],
+      },
     edit && {
       legend: _('Notification options'),
       fields: [
         'voicemailNotificationTemplate',
-        'faxNotificationTemplate',
+        faxEnabled && 'faxNotificationTemplate',
         'invoiceNotificationTemplate',
```

Comment:
invoiceNotificationTemplate seems like a field that could also be filtered by hasInvoicesFeature, but we can handle that in another PR if you prefer

Reply:
OK, i'll create an issue with that